.gitignore

```diff
@@ -5,7 +5,6 @@
 [Oo]bjcs
 [Oo]bjvb
 artifacts
-src/VersionInfo.cs

```

Comment:
I prefer to leave these. It makes for a cleaner sandbox if/when one switches between versions.

Reply:
OK, I'll put it back in.

Reply:
Actually, I think we should remove this. I just failed to build on my work machine because I still had this file, which was puzzling because git said my repo had no pending change. If VersionInfo.cs had been removed from .gitignore, I would have seen the extra file, which would have told me why the build failed.